src/main/java/com/bnbair/bnbair/controller/UserController.java

```diff
@@ -50,4 +50,15 @@ public ResponseEntity<Void> deleteUser(@PathVariable Long id) throws UserNotFoun
         userService.deleteUser(id);
         return ResponseEntity.noContent().build();
     }
+    @GetMapping("/search")
+    public ResponseEntity<List<User>> getUsersByFullNames(@RequestParam String firstName, @RequestParam String lastName) {
+        List<User> users = userService.getUsersByFirstAndLastName(firstName, lastName);
+        return ResponseEntity.ok(users);
+    }
+
+    @GetMapping("/findByEmail")
```

Comment:
Are we sure this is a endpoint we should expose? Security wise?

Reply:
yeah that was an endpoint i added for the sake of it - please suggest some other repository method if you can think of any!

Reply:
Then I guess it's good enough to have it as is. The assignment only requires the very basic requests.